Sources/Purchasing/Purchases/Purchases.swift

```diff
@@ -1101,6 +1101,18 @@ public extension Purchases {
 
 #endif
 
+    @available(iOS 15.0, macOS 12.0, tvOS 15.0, watchOS 8.0, *)
+    func handleObserverModeTransaction(
+        purchaseResult: StoreKit.Product.PurchaseResult
+    ) async throws -> StoreTransaction? {
```

Comment:
I know this is Observer Mode but does it matter what they have set for the StoreKit versioN? Like... should this also require `.with(storeKitVersion: .storeKit2)`?

Reply:
Good catch. Indeed that's a requirement.